<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>320</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>391</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="owner">
         <item>
          <property name="text">
           <string>(none)</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ownID">
         <property name="text">
          <string>Owner ID: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ownName">
         <property name="text">
          <string>Name: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ownSurname">
         <property name="text">
          <string>Surname: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ownMoney">
         <property name="text">
          <string>Owner Money: </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="building">
         <item>
          <property name="text">
           <string>(none)</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="buildName">
         <property name="text">
          <string>Building Name: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="buildAdd">
         <property name="text">
          <string>Building Address:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="buildPrice">
         <property name="text">
          <string>Building Price</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QComboBox" name="onw1Ch">
       <item>
        <property name="text">
         <string>(none)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Trade</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="bldCh">
       <item>
        <property name="text">
         <string>(none)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QComboBox" name="own2Ch">
       <item>
        <property name="text">
         <string>(none)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="balance">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QPushButton" name="trade">
       <property name="text">
        <string>Trade</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QLabel" name="status">
       <property name="text">
        <string>Status: </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="priceToBuy">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
